### DISCUSSION TITLE
Office_S_Tips/pki/2019/09/19/Dlugoterminowa_waznosc_e-podpisu

### DISCUSSION THREAD

**Discussion by giscus** [andrzejQ/Office_S_Tips#19] [Komentarze]
# Office_S_Tips/pki/2019/09/19/Dlugoterminowa_waznosc_e-podpisu

W sytuacji, gdy będzie potrzeba sprawdzania ważności podpisu także po okresie ważności certyfikatów powinniśmy używać opcji LTV (Long Term Validation).

https://andrzejq.github.io/Office_S_Tips/pki/2019/09/19/Dlugoterminowa_waznosc_e-podpisu.html

**Comment by andrzejQ** [comment]
Nowa wersja komentarzy z użyciem `giscus -> GitHub Discussions`.
